components/cdc/src/endpoint.rs

```diff
@@ -534,16 +534,24 @@ impl<T: 'static + RaftStoreRouter<RocksEngine>> Endpoint<T> {
 
         let (cb, fut) = tikv_util::future::paired_future_callback();
         let scheduler = self.scheduler.clone();
+        let observe_id = delegate.id;
         let deregister_downstream = move |err| {
```

Comment:
```suggestion
        let deregister = move |err| {
```

Because it may deregister a region.